README.md

```diff
@@ -7,6 +7,17 @@
 This project uses [Yarn](https://yarnpkg.com/getting-started) and [Volta](https://docs.volta.sh/guide/) for package management and version switching.
 So, make sure to install both to use the project.
 
+# Yarn Command Reference
+```sh
+yarn 
+```
+Installs all dependencies listed in the root package.json.
+
+If `yarn` is not installed 
+```sh
+npm install --global yarn
+```
+
```

Comment:
We have volta configured in the package.json, it would be required for a contributor to have it installed before making any contribution.

This will help us have common node & yarn versions across contributors.